scalameta/trees/shared/src/main/scala/scala/meta/internal/trees/ParentChecks.scala

```diff
@@ -131,7 +131,7 @@ object ParentChecks {
 
   private[meta] def MemberTuple(tree: Member.Tuple, parent: Tree, dest: String): Boolean = {
     tree.args match {
-      case Nil | Member.Tuple(_ :: Nil) => false
+      case Nil | (Member.Tuple(_ :: Nil) :: Nil) => false
```

Comment:
`tree.args` is a list so it couldn't match a single tuple. I added `:: Nil`

Reply:
the (new) compiler tells us this is effectively dead code, so to avoid changing the behavior, shouldn't it be removed instead of changed?

Reply:
I think that was actually a bug, the check was supposed to be if the tuple consist of a single element tuple, which is done now.

Reply:
For sure it looks like a bug, but I was thinking it might be safer to address it in a separate commit/PR, as this could introduce different behaviors for a given scalameta version depending on the full scala version (in case this commit is cherry-picked for back-publishing AND this code is actually used by semanticdb-scalac...).

Reply:
added as a separate commit in #3579.

Reply:
The problem is that I need to cherry pick that also for 4.9.0, since the exact same thing will fail while compiling :/